variant/regen.py

```diff
@@ -75,12 +75,7 @@
 INSERT INTO T VALUES ('primitive_binary', X'31337deadbeefcafe'::Variant);
 INSERT INTO T VALUES ('primitive_string', 'This string is longer than 64 bytes and therefore does not fit in a short_string and it also includes several non ascii characters such as 🐢, 💖, ♥️, 🎣 and 🤦!!'::Variant);
 
--- https://github.com/apache/parquet-testing/issues/79
--- is not clear how to create the following types using Spark SQL
--- TODO TimeNTZ                    (Type ID 17)
--- TODO 'timestamp with timezone (NANOS)'  (Type ID 18)
--- TODO 'timestamp with time zone (NANOS)' (Type ID 19)
--- TODO 'UUID'                     (Type ID 20)
+-- binary artifacts of 'TimeNTZ'/'timestamp with timezone (NANOS)'/'timestamp with time zone (NANOS)'/'UUID' was generated by the iceberg test code, please ref to https://github.com/apache/parquet-testing/pull/92 for more detail
```

Comment:
timestamp with timezone is duplicated?

Reply:
Ah, there is one for `without`, fixed it.